components/general/NetPyNEAddNew.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import ContentAdd from 'material-ui/svg-icons/content/add';
 import FloatingActionButton from 'material-ui/FloatingActionButton';
+require('./NetPyNEAddNew.less');
```

Comment:
I haven't started testing it properly but sth I have just noticed when compiling: this file is missing, probably you forgot to push it :)

Reply:
@adrianq thx for the spot, fixed, I was originally planning to use this but I did not need at the end.